src/resources/authoring/gv-1.json

```diff
@@ -2684,6 +2684,7 @@
     },
     "allele-targetMatch-visible-simpleColors" : {
       "challengeType" : "match-target",
+      "showDrakeColorHint": true,
```

Comment:
1. It appears that the authoring document has been auto-formatted to alphabetize properties. I have been maintaining this convention, even though I don't particularly agree with it. (I would prefer dialog sections to be in the order `start, middle, end`, for instance, rather than the reverse.) In this case, putting `showDrakeColorHint` next to the `showUserDrake` property doesn't seem unreasonable.
2. You should add this to the schema for the `FVGenomeChallenge`.